index.rst

```diff
@@ -15,5 +10,265 @@ Change Record
+
+.. _cycles-and-releases:
+
+Cycles and Releases
+~~~~~~~~~~~~~~~~~~~
+
+The work is executed through 6-month *cycles*. Scope of work for each cycle is driven by the Baseline
+Plan, in particular by milestones. Each cycle ends with a new software release. Releases are
+thoroughly tested and documented.
+
+Cycles are planned through *epics*. Each epic must be defined such that it can be completed
+within a single cycle.
+
+Once a plan is defined for a given cycle, it is loaded into PMCS and changed-controlled. A plan for
+a cycle is loaded to PMCS during the month proceeding the start of the cycle.
+
+Any non-trivial adjustments to the plan that affect scope, schedule or budget must be approved
+by CCB.
+
+It is acceptable to load the plan in 3-month chunks, e.g., the plan for
+the first 3 months of the cycle is loaded before the start of the cycle, and the remaining
+part of the plan covering the last 3 months is loaded before the 4th month starts. This
+allows for minor fine-tuning of the second half of the cycle without going through the CCB
+approval.
+
+Epics-based Long-term Planning
+------------------------------
+
+As explained above, epics are used for planning and executing work within a cycle.
+
+In addition to that, epics are also extremely valuable for longer-term planning at a fine-grain level.
+When details of work for a given planning package are known, they can and should be captured through
+epics. Such epics can be freely created and changed at any time without any approvals. They
+should, of course, fit within the scope and budget of related PP. They can be useful for
+bottom-up analysis and validation of resources needed to implement a given PP. This allows
+to do detailed planning in flexible and agile way, while ensuring the scope/cost/schedule is
+well controlled and managed.
+
+Level of Effort
+---------------
+
+Two general rules are applied for the DM Team labor when determining whether an activity should be
+treated as LOE or non-LOE:
+
+* All activities that have a clear deliverable are tracked through non-LOE. Examples of non-LOE
+  activities include writing a new piece of software, purchasing new equipment, or adding new
+  machines to the cluster.
+
+* Activities that can not be predicted, as well as short, recurring routine activities are considered LOE.
+  Examples of LOE activities include replacing faulty disk drive, cluster maintenance, or writing monthly
+  status report.
+
+For planning purposes, 70% of available time of each software developer is assumed to be available
+for "pure, uninterrupted coding", and the remaining 30% is considered as "overhead", and tracked
+explicitly as LOE.
+
+A small fraction of all DM labor is performed by Contractors. The contractor labor is managed as LOE,
+directly in dollars.
+
+People
+======
+
+* DM Project Manager responsible for overall plan, schedule and priorities.
+
+* Work divided into smaller pieces, through WBS.
```

Comment:
oops, this whole section is currently just a set of bullets, I was planning to turn it into real text and I somehow missed it.